Frameworks/CoreFoundation/PlugIn.subproj/CFBundle_Resources.c

```diff
@@ -46,6 +46,19 @@
 #include <sys/stat.h>
 #include <errno.h>
 
+#if WINOBJC
+// WINOBJC: redirect more things to work with Ebr
+// since unfortunately that is still needed
+#define close _NS_close
+#define write _NS_write
+#define read _NS_read
+#define open _NS_open
+#define stat(a,b) _NS_stat64i32(a,b)
+#define fstat _NS_fstat
+#define mkdir(a,b) _NS_mkdir(a)
+#define rmdir _NS_rmdir
+#define unlink _NS_unlink
```

Comment:
It worries me that these are scatted across so many files, but I see that that is a la mode